Examples/SampleApp/.gitignore

```diff
@@ -21,6 +21,14 @@ DerivedData
 *.ipa
 *.xcuserstate
 
+# Android/IJ
+#
+.idea
+.gradle
+local.properties
+*.iml
```

Comment:
I've seen many android OSS projects include iml files so I'd rather leave it out as it's more like a personal choice whether to include them or not.